src/command_line_parser.cc

```diff
@@ -1912,8 +1923,21 @@ CLParser::ParseCommandLine(int argc, char** argv)
   }
 
   // In fixed schedule mode, the request count is based on the number of
-  // payloads in the user-provided dataset
+  // payloads in the user-provided dataset (minus warmup request count)
   if (params_->inference_load_mode == pa::InferenceLoadMode::FixedSchedule) {
+    if (params_->request_count != 0) {
+      throw std::runtime_error(
+          "Cannot specify `--request-count` when using `--fixed-schedule`. The "
+          "request count will be equal to the number of requests in the input "
+          "data minus the warmup request count.");
+    }
+
+    if (params_->user_data.empty()) {
+      throw std::runtime_error(
+          "When using `--fixed-schedule`, you must provide a dataset using "
+          "`--input-data`.");
+    }
+
```

Comment:
Just some UX cleanup/validation.

PA now forbids user from specifying `--request-count` in `--fixed-schedule` mode.

Fixed schedule mode deduces request count based on the dataset size minus the specified warmup request count.